freeimage/bld.bat

```diff
@@ -0,0 +1,12 @@
+
+if defined ProgramFiles(x86) (
+  set BDIR=Dist\x64
+) else (
+  set BDIR=Dist\x32
+)
+
+copy %BDIR%\FreeImage.dll %LIBRARY_BIN%\FreeImage-3.17.0.dll
+copy %BDIR%\FreeImage.dll %LIBRARY_BIN%\FreeImage.dll
+copy %BDIR%\FreeImage.h %LIBRARY_INC%\FreeImage.h
+copy %BDIR%\FreeImage.lib %LIBRARY_LIB%\FreeImage.lib
+copy C:\Windows\System32\vcomp120.dll %LIBRARY_BIN%\vcomp120.dll || exit "needs VC redist from http://www.microsoft.com/en-us/download/details.aspx?id=40784"
```

Comment:
:disappointed: You shouldn't do that! Please see Steve Dower's [excellent discussion on why you shouldn't mix Vistual Studio CRTs](http://stevedower.id.au/blog/building-for-python-3-5/). Luckily, we can solve this problem - have a look at my [conda recipe for freeimage](http://www.github.com/menpo/conda-freeimage).